### PR TITLE
Scripts for release management

### PR DESCRIPTION
**Pull request by fisx**
- remove experimental code for creating python eggs
- bump versions
- create tar balls with binaries

### CODE REVIEW

**Comment by fisx** [comment]
The travis error is related to the purescript frontend and can be ignored here; work-around is included in #448.

**Comment by fisx** [comment]
- pushed release management instructions as discussed offline
- travis should be happy now.  let's see...

File: misc/release/thentos-captcha-README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Thentos Captcha
```
Comment (ChristianSi) [inline misc/release/thentos-captcha-README.md]: I propose to move this document into the `docs` folder (and adapt `thentos-captcha-release.sh` accordingly).

**Comment by ChristianSi** [comment]
Looks good. @fisx, please address my comment (or not) and merge.

**Pull request merged**